gitbooks/features/token-compression.zh-CN.md

```
@@ -0,0 +1,51 @@
+
+每条规则命名一个工具/命令模式和一个压缩策略（截断、行去重、折叠空白、删除匹配的正则表达式、摘要分段等）。新规则就是 JSON 文件，无需重新编译。
+
+## 为什么这和记忆有关
+
+TokenJuice 是使[自动拉取](obsidian-wiki/auto-fetch.md)在经济上可行的原因。当 Gmail provider 同步一页 200 条消息时，TokenJuice 在每个规范化的邮件进入构建摘要的模型**之前**就将其压缩。GitHub diff、Slack 频道转储以及其他任何高流量来源同理。
+
+具体来说：通过前沿模型摄入你最近六个月的邮件费用从数百美元降到个位数美元。
+
+## 它在流水线中的位置
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)